sdk/core/core-http/lib/webResource.ts

```diff
@@ -503,6 +503,7 @@ export interface RequestOptionsBase {
 
   /**
    * The number of milliseconds a request can take before automatically being terminated.
+   * If the request is terminated, an abort error is thrown with the message - "The operation was aborted."
```

Comment:
Is it relevant to mention the actual error message?

```suggestion
   * If the request is terminated, an `AbortError` is thrown.
```

Reply:
I don't know, I wasn't sure if the `AbortError` is standard enough.

Reply:
Applied your suggestion.